Examples/HackerNewsClient/HackerNewsClient/UI/Browser.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  Browser.swift
+//  HackerNewsClient
+//
+//  Copyright © 2016 Bending Spoons.
+//  Distributed under the MIT License.
+//  See the LICENSE file for more information.
+
+import Foundation
+import Katana
+import KatanaElements
+
+struct Browser: NodeDescription, PlasticNodeDescription {
```

Comment:
I believe there should be a spinning wheel when fetching the content. 

One way to achieve this is probably to move the loading of the content in an AsyncAction, another way would be to pass the update() function to the nativeView. I'd probably go for the first.